app/views/expenses/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="top_bar">
+  <%= render partial: 'shared/menu', locals: { title: 'EXPENSE DETAIL', menu_path: root_path } %>
+</div>
+
+<section class="group_detail_page">
+  <div class="group">
+    <div>
+      <% if @expense.icon.attached?%>
+        <%= image_tag(@expense.icon) %>
+      <%else%>
+        <%= image_tag('expense.png') %>
+      <%end%>
+      <h2><%= @expense.name %></h2>
+    </div>
+    <p class="desc">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque iste quidem repudiandae. Dolor debitis quas impedit sit officiis dolorum beatae quaerat nesciunt doloribus expedita in distinctio laborum facilis ex sunt ipsum numquam, amet et odio iusto ab est. Vero, nemo.</p>
```

Comment:
- Kudos working on the details page for the category and the transaction page 🔥💯✅🤩🤩🤩